Assignment5/Assignment5_QNo3/SolAssignmentNo5_QNo3.rb

```diff
@@ -0,0 +1,22 @@
+# => 3. Write a program to define a method inside a module and
+# => then use the module in such a way that the method can be
+# => accessible as an instance method of that class.
+
+# => import the module file
+require_relative 'module_instance_method.rb'
+
+class ClassModuleInterface
+  # => include both modules
+  include ModuleDemoInstanceMethod
+
+  # => call the instance_method_behave
+  def call_instance_method
+    puts "The addition from Module Add is = "
+    method_instance_behave
+  end
```

Comment:
@HrisikeshSinghaChowdhury You can use those module methods directly as instance methods

Reply:
ok

Reply:
done